iree/compiler/Conversion/LinalgToSPIRV/KernelDispatchUtils.cpp

```diff
@@ -83,13 +83,65 @@ static LogicalResult getOpLaunchConfig(T op, const spirv::TargetEnv &targetEnv,
   return op.emitError("undefined launch config for tiled operation");
 }
 
+static void getMaliBestMatMulTileSizes(Type elementType,
+                                       SmallVectorImpl<int64_t> &tileSizes) {
+  if (elementType.isF16()) {
+    tileSizes.append({16, 64, 8});
+  } else {
+    tileSizes.append({8, 64, 4});
+  }
+}
+
+/// Launch configuration for different known GPU configuration.
+static LogicalResult getTargetSpecificConfig(
```

Comment:
Since this is specific to Mali, can we rename this function to `getMali...Config`?